apt-spy-2-bootstrap.sh

```diff
@@ -43,13 +43,12 @@ if [ "$(gem search -i apt-spy2)" = "false" ]; then
   echo "Installing apt-spy2 (and prerequisites)..."
   gem install --no-ri --no-rdoc apt-spy2
   echo "... apt-spy2 installed!"
-  echo "... Setting 'apt' sources.list for closest mirror to country=$COUNTRY"
-  apt-spy2 fix --launchpad --commit --country=$COUNTRY
-else
-  echo "... Setting 'apt' sources.list for closest mirror to country=$COUNTRY"
-  apt-spy2 fix --launchpad --commit --country=$COUNTRY
 fi
 
+echo "... Setting 'apt' sources.list for closest mirror to country=$COUNTRY"
+apt-spy2 check
+sudo apt-spy2 fix --commit
```

Comment:
I do not recommend this change, we go through a lot of trouble to look up the country code, it would be a shame to not use it. In our experimentation, running Vagrant-DSpace in other countries, the previous command line (apt-spy2 fix --launchpad --commit --country=$COUNTRY) worked more reliably. Putting a sudo in front of that is a *great* idea, though. A recent refactor of how we call shell provisioners necessitated this change, but we missed this detail. Good catch!